### DISCUSSION TITLE
Ability To Set Control Buttons Size

### DISCUSSION THREAD

**Discussion by fire1ce** [Selbi182/SpotifyBigPicture#27] [Ideas]
Hey @Selbi182 nice job on the project! 
I've found your project very nice as a mini "control panel" for spotify. 
Below an image of Raspberry pi 4 with 7 inch touch screen with your SpotifyBigPicture. I love it.
The only thing i find hard is to hit the play/next... buttons. 
Is there a way to add an option for size and spacing of the buttons? Thanks in advance!  

![image](https://github.com/Selbi182/SpotifyBigPicture/assets/16795594/0ed65dec-f1cb-4815-b9cc-8b03abbe8ed2)

**Comment by Selbi182** [comment]
Hey, thank you so much for your kind words!

I see the issue. To be fair, I never use the controls myself so I wasn't really aware of any usability issues, but this is definitely a fair point. I'll see what I can do for the next update! :)

A quick mockup, would this be big enough?
![grafik](https://github.com/Selbi182/SpotifyBigPicture/assets/8850085/ed428fd2-cd4a-49d9-a102-1fa7709d53d1)

**Reply by fire1ce** [reply]
much appreciated!
yeh it looks better for touch use. can i suggest increase the margin between the button a bit?

**Reply by Selbi182** [reply]
Sure, I'll tweak the design a bit. I was just looking for feedback on the button size itself.

**Comment by Selbi182** [comment]
Hey, sorry for the long silence, I haven't really worked on the project in quite some time. I just wanted to ask if this is still relevant?

**Comment by fire1ce** [comment]
Heh yeh i know it’s not easy and also time consuming to support open source projects. Any way, it will be nice addition but… only when you will have the mood and the motivation to do it.

**Reply by Selbi182** [reply]
Alright, thanks for letting me know!

**Comment by JosephIris** [comment]
+1 to @fire1ce's request (funny thing is that he and I knew each other like a decade and a half ago and both of us wanted to use SpotifyBigPicture with a touch interface!)

**Comment by Selbi182** [comment]
Sorry for the long wait, but this has now finally been resolved. I've reworked playback controls entirely for the latest release. Still figuring out why Windows Defender suddenly doesn't like my software anymore, but if you wanna give it a shot anyway, see here: 

https://github.com/Selbi182/SpotifyBigPicture/releases/tag/v1.13.0
